packages/modules/uniswap-v3/src/actions/__mock__/utils.ts

```diff
@@ -65,61 +76,46 @@ export const setupUniswapV3 = async (): Promise<UniswapV3Registry> => {
   const rich_man_client = new TestInfinitWallet(TestChain.arbitrum, '0xF977814e90dA44bFA03b6295A0616a897441aceC')
   await rich_man_client.sendTransactions([{ name: 'transfer', txData: transferTx }])
   // approve
-  let approveData = encodeFunctionData({
+  const approveNpmData = encodeFunctionData({
     abi: erc20Artifact.abi,
     functionName: 'approve',
     args: [curRegistry.nonfungiblePositionManager!, 115792089237316195423570985008687907853269984665640564039457584007913129639935n],
   })
-  await client.walletClient.sendTransaction({
-    data: approveData,
-    to: usdt,
-  })
-  await client.walletClient.sendTransaction({
-    data: approveData,
-    to: weth,
-  })
-  approveData = encodeFunctionData({
+  const approveSwapRouterData = encodeFunctionData({
     abi: erc20Artifact.abi,
     functionName: 'approve',
     args: [curRegistry.swapRouter02!, 115792089237316195423570985008687907853269984665640564039457584007913129639935n],
   })
-  await client.walletClient.sendTransaction({
-    data: approveData,
-    to: usdt,
-  })
-  await client.walletClient.sendTransaction({
-    data: approveData,
-    to: weth,
-  })
-  const uniswapV3FactoryArtifact = await readArtifact('UniswapV3Factory')
+
   // create new pool
   const createPoolData = encodeFunctionData({
     abi: uniswapV3FactoryArtifact.abi,
     functionName: 'createPool',
     args: [weth, usdt, 500],
   })
-  await client.walletClient.sendTransaction({
-    data: createPoolData,
-    to: curRegistry.uniswapV3Factory!,
-  })
+
+  await client.sendTransactions([
+    { name: 'approve USDT to NPM', txData: { data: approveNpmData, to: usdt } },
+    { name: 'approve WETH to NPM', txData: { data: approveNpmData, to: weth } },
+    { name: 'approve USDT to SwapRouter', txData: { data: approveSwapRouterData, to: usdt } },
+    { name: 'approve WETH to SwapRouter', txData: { data: approveSwapRouterData, to: weth } },
+    { name: 'Create new pool', txData: { data: createPoolData, to: curRegistry.uniswapV3Factory } },
```

Comment:
Change the name of each tx to be capitalized or lowercase text to make it consistent.?